data/hai/hai reveal 5 philosophers.txt

```diff
@@ -533,9 +642,12 @@ mission "Hai Reveal [B06-long] Boarded Smuggler"
 		government "Smuggler (Hai Trafficker)"
```

Comment:
You need to use continuous weapons, and hit smugglers that are not near other ships.

Changes to governments.txt are outside the scope of this PR. That would have to be in a separate smuggler balance PR.

Reply:
You should either open an issue for this, or do a PR with balance changes.

Reply:
I think the solution is to:

1. Remove all affinities to the Smugglers from governments.txt except for pirates since nobody is supposed to be able to identify these except pirates.
2. Give all Smuggler fleets the "target" personality so they'll be labeled "mission target" and blink on the radar, just like any other mission target.

Reply:
In #7734, the Smugglers will be labeled as mission targets. It only solves half the problem; the Navy might blow them up before you get to them. You'll be able to find them first though.